nexus/flow-rs/src/grpc.rs

```diff
@@ -176,6 +176,7 @@ impl FlowGrpcClient {
             script: job.script.clone(),
             system: system as i32,
             idle_timeout_seconds: job.sync_interval.unwrap_or_default(),
+            disable_peerdb_columns: job.disable_peerdb_columns,
```

Comment:
so like ui flag here is that while bool may belong in model, by time it hits config we should clear fields instead of further propagation of this bool

Reply:
makes sense, I had it at the handler level initially but moved it here in case someone tries to set it for existing flows and also simplify testing

Reply:
for testing you can set the existing fields. same goes for the bool tracking deleted at being there, that's a ui flag, on proto empty string signals